File: Monstone_Android_Client/app/src/main/java/com/example/nightwingky/monstone/main_program/fragmentHome/fresh/FreshViewHolder.java
package com.example.nightwingky.monstone.main_program.fragmentHome.fresh;

import android.view.View;
import android.widget.TextView;

import com.example.nightwingky.monstone.R;
import com.example.nightwingky.monstone.main_program.fragmentHome.TypeAbstractViewHolder;

/**
 * Created by nightwingky on 17-3-18.
 */

public class FreshViewHolder extends TypeAbstractViewHolder<Object> {

    private TextView tv_fresh;

    public FreshViewHolder(View itemView) {
        super(itemView);

        this.tv_fresh = (TextView) itemView.findViewById(R.id.fresh_home_list);
    }

    @Override
    public void bindHolder(Object object) {

    }
}

File: kotlin_client/app/src/main/java/com/graduation/nightwingky/book_drifting/monstone/main_program/fragmentHome/fresh/FreshViewHolder.kt
package com.graduation.nightwingky.book_drifting.monstone.main_program.fragmentHome.fresh

import android.view.View
import android.widget.TextView

import com.graduation.nightwingky.book_drifting.R
import com.graduation.nightwingky.book_drifting.monstone.main_program.fragmentHome.TypeAbstractViewHolder

/**
 * Created by nightwingky on 17-3-18.
 */

class FreshViewHolder(itemView: View) : TypeAbstractViewHolder<Any>(itemView) {

    private val tv_fresh: TextView

    init {

        this.tv_fresh = itemView.findViewById<View>(R.id.fresh_home_list) as TextView
    }

    override fun bindHolder(`object`: Any) {

    }
}
